### PR TITLE
Support exposing jmx and additional ports

### PR DESCRIPTION
**Pull request by ethanocentricity**
Allow the option of exposing a jmx port and additional ports from the broker container to support monitoring. For example, kafka manager access the jmx metrics directly. Additional ports can be used to expose a prometheus exporter endpoint (when using a docker container that includes the jmx exporter as a java agent for kafka as described here: https://www.robustperception.io/monitoring-kafka-with-prometheus/).

When exposing additional ports, one would presumably create a service to access those ports, but it seemed best to keep that outside of this helm chart since it would complicate things with more conditionals and adding more values related to the additional service ports. It could be done, but it seemed wiser to just stay simple for now.

(PodPresets couldn't be used to add on the desired functionality since the ports are at the container level, not the pod level.)

### CODE REVIEW

File: incubator/kafka/README.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ following configurable parameters:
 | `external.domain`              | Domain in which to advertise Kafka external listeners.                                                          | `cluster.local`                                            |
 | `external.init`                | External init container settings.                                                                               | (see `values.yaml`)                                        |
 | `configurationOverrides`       | `Kafka ` [configuration setting][brokerconfigs] overrides in the dictionary format                              | `{ offsets.topic.replication.factor: 3 }`                  |
+| `jmxPort`                      | The jmx port to use for brokers.  Will expose both the port and set the KAFKA_JMX_PORT env variable.            | blank                                                      |
+| `additionsPorts`               | Additional ports to expose on brokers.  Useful when the image exposes metrics (like prometheus, etc.) through a javaagent instead of a sidecar   | `{}`                                  |
```
Comment (benjigoldberg) [inline incubator/kafka/README.md]: `additionsPorts` -> `additionalPorts`

File: incubator/kafka/values.yaml [hunk 2]
```diff
@@ -83,6 +83,14 @@ configurationOverrides:
...
   #   PLAINTEXT:PLAINTEXT,EXTERNAL:PLAINTEXT
 
+## The jmx port to use for brokers.  Will expose both the port and set the KAFKA_JMX_PORT env variable
+# jmxPort: 9998
+
+## A collection of additional ports to expose on brokers (formatted as normal containerPort yaml)
+# Useful when the image exposes metrics (like prometheus, etc.) through a javaagent instead of a sidecar
+additionalPorts: {}
+
+
```
Comment (benjigoldberg) [inline incubator/kafka/values.yaml]: rm extra newline

**Review by benjigoldberg — COMMENTED** [review]
@ethanocentricity couple of minor formatting things, otherwise this looks great.

**Comment by benjigoldberg** [comment]
@ethanocentricity if you take care of those handful of comments I made and up the chart version I would be happy to merge this change 👍

**Comment by ethanocentricity** [comment]
@benjigoldberg Thanks for reminding me...  Hopefully it is good to go now.

**Pull request merged**